src/Apps/Artist/Components/__tests__/ArtistHeader.test.tsx

```diff
@@ -39,4 +39,16 @@ describe("ArtistHeader", () => {
     expect(html).toContain(">Following<")
     expect(html).not.toContain(">Follow<")
   })
+
+  it("renders blue chip indicator", async () => {
+    const wrapper = await getWrapper()
+    const html = wrapper.html()
+    expect(html).toContain("Blue chip")
+  })
+
+  it("renders auction record indicator", async () => {
+    const wrapper = await getWrapper()
+    const html = wrapper.html()
+    expect(html).toContain("Auction Record")
+  })
```

Comment:
Can you add tests verifying that if the correct data isn't present, it does not show these indicators?